universal_transfer_operator/src/universal_transfer_operator/data_providers/filesystem/sftp.py

```diff
@@ -125,6 +132,17 @@ def write_using_smart_open(self, source_ref: FileStream) -> str:
             stream.write(source_ref.remote_obj_buffer.read())
         return complete_url
 
+    def write_dataframe_using_smart_open(self, source_ref: pd.DataFrame) -> str:
```

Comment:
Have we tested this?

Reply:
Yup, this test case checks for everything - https://github.com/astronomer/astro-sdk/blob/8dc5da8d7ccdc5ecd665bba70434e2fc4c8b831f/universal_transfer_operator/tests_integration/test_data_provider/test_databases/test_base.py#L75 
Do you have anything besides this you want to test?

Reply:
It will be covered as part of the this integration test - https://github.com/astronomer/astro-sdk/blob/8dc5da8d7ccdc5ecd665bba70434e2fc4c8b831f/universal_transfer_operator/tests_integration/test_data_provider/test_databases/test_base.py#L75
But I have also added test to check specifically that method.